modules/azurerm/Firewall-NAT-Rules/firewall_nat_rules.tf

```diff
@@ -0,0 +1,41 @@
+# -------------------------------------------------------------------------------------
```

Comment:
Shall we rename the module to `Firewall-LB-DNAT-Rule-Collection` as this specifically adds DNAT rules at firewall for internal LB.